cmd/api-resource-collector/log.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright © 2020 Red Hat Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package main
+
+import (
+	"fmt"
+	"os"
+)
+
+var debugLog bool
+
+func LOG(format string, a ...interface{}) {
```

Comment:
I know we talked about debug logging for a long time and this is as good of a start as any. Do you think it would make sense to move this file to a shared module used by all the helper containers? If not, we should at least note this file in a ticket (I think we already have one?)

Reply:
let's do a follow up for this as there's probably something a little cleaner we can do. This one I used here mostly to save typing.